File: doc/src/codes/Users_Manual/gain/holo_0.py
from pyautd3.gain.holo import GSPAT, NalgebraBackend, pascal

backend = NalgebraBackend()
g = (
    GSPAT(backend)
    .add_focus([x1, y1, z1], 5e3 * pascal)
    .add_focus([x2, y2, z2], 5e3 * pascal)
)

File: doc/src/codes/Users_Manual/gain/holo_0.cs
using AUTD3Sharp.Gain.Holo;
using AUTD3Sharp.Utils;
using static AUTD3Sharp.Gain.Holo.Amplitude.Units;

var backend = new NalgebraBackend();
var g = new GSPAT<NalgebraBackend>(backend)
            .AddFocus(new Vector3d(x1, y1, z1), 5e3 * Pascal)
            .AddFocus(new Vector3d(x2, y2, z2), 5e3 * Pascal);